src/routers/expenses.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const express = require('express');
+const expenseServices = require('../controllers/expenses');
+
+const router = express.Router();
+
+router.get('/', expenseServices.getAll);
+router.get('/:expenseId', expenseServices.getOne);
+router.post('/', expenseServices.create);
+router.patch('/:expenseId', expenseServices.update);
+router.delete('/:expenseId', expenseServices.remove);
+
+module.exports = { router };
```

Comment:
Instead of exporting an object with a 'router' property, you can directly export the router.